src/validator.ts

```diff
@@ -30,7 +30,13 @@ export class Validator {
     } else {
       validate = ajv.compile(schema);
     }
-    if (validate === undefined) return null;
+    if (validate === undefined) {
+      if (typeof schema === 'string') {
+        throw new Error(`Unregistered schema requested: '${schema}'`);
+      } else {
+        throw new TypeError(`Error compiling schema`);
+      }
```

Comment:
This line is not covered by tests.

Reply:
Also unnecessary use of a template literal.

Reply:
This should both be fixed on the next commit